### ISSUE TITLE
Create launcher for ShuffleBoard for Driver Station

### ISSUE THREAD

**Issue by abrightwell** [wando-advanced-robotics/PowerUpRobot#13]
According to the Shuffleboard [README](https://github.com/wpilibsuite/shuffleboard/blob/master/README.md) there are two ways to start up Shufleboard.

1. Via eclipse plugin.
2. By manually running `java -jar <path/to/wpilib>/tools/Shuffleboard.jar`

Neither of these options are convenient nor viable for competition.

So, let's either:

1. Identify a solution in the community that addresses easily opening Shuffleboard.

or,

2. Create a simple script/batch file that can be used to open Shuffleboard.

This solution needs to be implemented on the official driver station computer.

**Comment by jssjr** [comment]
Gradle-RIO has a `shuffleboard` target that should manage launching this similar to a batch file. I don't have this configured on my laptop here to test, but may be a viable option. See https://github.com/Open-RIO/GradleRIO/blob/9f0f2dadfd38519f611229a193288d0c69746525/README.md#commands for an overview.

**Comment by abrightwell** [comment]
You are right. However, my thought is that the Driver Station is purely the driver station without any dev tools on it. Having something that can launch it from the desktop via a double-click is what I'm thinking here.
